Added alternative way to adding terms of use link (#95657)
Azure/azure-content: articles/active-directory-b2c/javascript-and-page-layout.md

In the code, replace `termsOfUseUrl` with the link to your terms of use agreement. For your directory, create a new user attribute called **termsOfUse** and then include **termsOfUse** as a user attribute.

> Alternatively, you can add a link to the bottom of self-asserted pages, without the use of Javascript, using the following localization.

Reviewer: ```suggestion
Alternatively, you can add a link at the bottom of self-asserted pages, without using of JavaScript. Use the following localization:
```

```xml
<LocalizedResources Id="api.localaccountsignup.en">
  <LocalizedStrings>
    <!-- The following elements will display a link at the bottom of the page. -->
    <LocalizedString ElementType="UxElement" StringId="disclaimer_link_1_text">Terms of use</LocalizedString>
    <LocalizedString ElementType="UxElement" StringId="disclaimer_link_1_url">termsOfUseUrl</LocalizedString>
    </LocalizedStrings>
</LocalizedResources>
```

> In the code, replace `termsOfUseUrl` with the link to your organization's privacy statement and terms and conditions. 

Reviewer: ```suggestion
Replace `termsOfUseUrl` with the link to your organization's privacy policy and terms of use. 
```
